sympy/integrals/intpoly.py

```diff
@@ -28,12 +30,15 @@ def polytope_integrate(poly, expr, dims=None):
     >>> polytope_integrate(poly, expr)
     1/4
     """
-    if dims is None:
+    if kwargs['dims'] is None:
         if isinstance(expr, Expr):
             dims = tuple(expr.free_symbols)
         else:
             dims = (x, y)
 
+    if kwargs['clockwise'] is True:
```

Comment:
Ah yes. I've made the change locally. There's a deficiency in the clockwise sort so I'll push the total changes after correcting that.